modules/apps/app-builder/app-builder-lang/src/main/resources/content/Language.properties

```diff
@@ -48,7 +48,7 @@ filter-entries=Filter Entries
 filter-entries-by-columns=Filter Entries by Columns
 filter-entries-by-setting-a-value-for-this-column=Filter entries by setting a value for this column.
 filter-entries-by-setting-filters-for-each-column=Filter entries by setting filters for each column.
-for-all-forms-of-this-object=For All Forms of This object
+for-all-forms-using-this-field=For all forms using This Field
```

Comment:
@jeyvison do you know why the `This` is capitalized after run `buildLang`?

Reply:
@interaminense I think it's SF that is doing this when the keys are generated. It happened in a old PR that Keven sent and we forwarded like this. Brian was fine with it.

Reply:
thanks, @jeyvison